src/Categories.jsx

```diff
@@ -0,0 +1,15 @@
+export default function Categories({ categories }) {
+  if (!categories.length) {
+    return null;
+  }
+
+  return (
+    <ul>
+      {categories.map((category) => (
+        <button key={category.id} type="button">
+          {category.name}
+        </button>
+      ))}
+    </ul>
```

Comment:
`ul`태그를 사용하는데, `li`태그를 사용하지 않는 것 같네요. `ul`태그를 사용하지 않아도 될 것 같아요.

```suggestion
    <ul>
      {categories.map(({ id, name }) => (
        <button key={id} type="button">
          {name}
        </button>
      ))}
    </ul>
```

구조 분해 할당으로 간략하게 표현하면 좋을 것 같아요